chord_metadata_service/metadata/settings.py

```diff
@@ -366,4 +368,16 @@ def get_secret(path):
     "SWAGGER_UI_SETTINGS": {
         "docExpansion": "none",  # collapse all endpoints by default
     },
+    # Specify Enum names for choices used by multiple fields
+    "ENUM_NAME_OVERRIDES": {
+        "BooleanEnum":  val.BOOLEAN,
+        "TCategoryEnum": val.T_CATEGORY,
+        "NCategoryEnum": val.N_CATEGORY,
+        "MCategoryEnum": val.M_CATEGORY,
+        "StageGroupEnum": val.STAGE_GROUP,
+        "StagingSystemEnum": val.TUMOUR_STAGING_SYSTEM,
+        "ReferencePathologyEnum": val.CONFIRMED_DIAGNOSIS_TUMOUR,
+        "MarginTypesEnum": val.MARGIN_TYPES
+    }
+
```

Comment:
We shouldn't import files to settings to avoid circular import errors (file import from settings and setting import from file). A better way to do this is:
```
 "ENUM_NAME_OVERRIDES": {
        "BooleanEnum": "chord_metadata_service.mohpackets.permissible_values.BOOLEAN",
        "TCategoryEnum": "chord_metadata_service.mohpackets.permissible_values.T_CATEGORY",
        "NCategoryEnum": "chord_metadata_service.mohpackets.permissible_values.N_CATEGORY",
        "MCategoryEnum": "chord_metadata_service.mohpackets.permissible_values.M_CATEGORY",
        "StageGroupEnum": "chord_metadata_service.mohpackets.permissible_values.STAGE_GROUP",
        "StagingSystemEnum": "chord_metadata_service.mohpackets.permissible_values.TUMOUR_STAGING_SYSTEM",
        "ReferencePathologyEnum": "chord_metadata_service.mohpackets.permissible_values.CONFIRMED_DIAGNOSIS_TUMOUR",
        "MarginTypesEnum": "chord_metadata_service.mohpackets.permissible_values.MARGIN_TYPES",
    },
```

Reply:
I don't see an issue here since there is no reason to ever import settings into permissible_values

Reply:
This is also best practice so please follow it.